test/e2e/providers/aws/aws.go

```diff
@@ -155,6 +155,8 @@ func getLatestWindowsAMI(ec2Client *ec2.EC2) (string, error) {
 			latestTime = newTime
 		}
 	}
+	// Make sure that image obtained is compatible with the test container image -
+	// "mcr.microsoft.com/powershell:lts-nanoserver-1809"
```

Comment:
The comment should be where `windowsAMIFilterValue` is being defined. You need to call out that if that value is being changed then test container image also needs to be updated.